client/src/components/DatasetInformation/DatasetError.vue

```diff
@@ -43,7 +43,7 @@
                         <br />
                         Start here:
                         <b>
-                            <a href="https://galaxyproject.org/support/tool-error/" target="_blank">
+                            <a href="https://training.galaxyproject.org/training-material/faqs/galaxy/#troubleshooting-errors" target="_blank">
```

Comment:
@hexylena Is this a better URL?

```suggestion
                            <a href="https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting.html" target="_blank">
```

Reply:
I considered that -- but I wanted users to land in a place that had other faqs. The direct link doesn't give that context, and the faqs cannot be searched yet, so the direct page is a dead-end. Interested in what @hexylena thinks though

Reply:
I think it makes sense, especially if the area around it becomes populated in the future with more FAQs. And it's a good point about very much being a 'dead end'. 

There are solutions to that, if we recommend more "related FAQs" on that page, or breadcrumbs to get back to the FAQ category, but, overall i think it's fine to just point to that page and revise later if the per-faq page becomes more exciting.

Reply:
Agree with all of that, thanks! Let's use the original link with context for now, and think about how to add breadcrumbs. This is the first of several links within Galaxy to update. We can always change our mind later.